ocean/global_ocean/WC_SO_Atl10/template_initial_state.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<template>
+	<namelist>
+		<option name="config_global_ocean_topography_smooth_iterations">10</option>
+		<option name="config_global_ocean_topography_smooth_weight">0.92</option>
```

Comment:
Not sure we want topography smoothing.

Reply:
My thinking is we probably do, but for consistency, I'd like to stick with whatever we did for the WC14 mesh.

Reply:
Okay, I'll check on that.

Reply:
WC14 didn't include any smoothing, but the SORRM mesh did.  For now, I have taken out smoothing to be consistent with WC14, but let me know if you'd prefer to be consistent with SORRM.

Reply:
After pondering this a bit more, I'd say we put our best configuration forward and I think that would include smoothing.  Let's switch to consistent with SORRM.  Sorry for changing again.

Reply:
No problem, that's why I asked.